wys/api/sql/mat-view-stationary-signs.sql

```diff
@@ -3,32 +3,57 @@ DROP MATERIALIZED VIEW wys.stationary_signs CASCADE;
 CREATE MATERIALIZED VIEW wys.stationary_signs
 TABLESPACE pg_default
 AS
- WITH distinctish_signs AS (
-         SELECT DISTINCT ON (api_id, (regexp_replace(sign_name, '([0-9]{5,8})'::text, ''::text))) api_id,
-            address,
-            regexp_replace(sign_name, '([0-9]{5,8})'::text, ''::text) AS sign_name,
-	 		id as sign_id,
-            dir,
-            start_date,
-	 		substring(sign_name from '([0-9]{5,8})'::text) as serial_num,
-            st_setsrid(st_makepoint(split_part(regexp_replace(loc, '[()]'::text, ''::text, 'g'::text), ','::text, 2)::double precision, split_part(regexp_replace(loc, '[()]'::text, ''::text, 'g'::text), ','::text, 1)::double precision), 4326) AS geom
-           FROM wys.locations
-          WHERE length("substring"(reverse(sign_name), '([0-9]{1,8})'::text)) > 3
-          ORDER BY api_id, (regexp_replace(sign_name, '([0-9]{5,8})'::text, ''::text)),
-	  				start_date
-        )
- SELECT api_id,
-     sign_id,
-    address,
-    sign_name,
-    dir,
-    start_date,
-	serial_num,
-    lead(start_date) OVER w AS next_start,
-    lag(start_date) OVER w AS prev_start,
-    geom
-   FROM distinctish_signs
-   WINDOW w as (PARTITION BY api_id order by start_date);
+WITH distinctish_signs AS (
+    SELECT DISTINCT ON (
+        loc.api_id, 
+        regexp_replace(loc.sign_name, '([0-9]{5,8})'::text, ''::text)
+    )
+        loc.api_id,
+        loc.address,
+        loc.id AS sign_id,
+        loc.dir,
+        loc.start_date,
+        regexp_replace(loc.sign_name, '([0-9]{5,8})'::text, ''::text) AS sign_name,
+        substring(loc.sign_name FROM '([0-9]{5,8})'::text) AS serial_num,
+        st_setsrid(
+            st_makepoint(
+                split_part(
+                    regexp_replace(loc.loc, '[()]'::text, ''::text, 'g'::text),
+                    ','::text, 2)::double precision, 
+                split_part(
+                    regexp_replace(loc.loc, '[()]'::text, ''::text, 'g'::text),
+                    ','::text, 1)::double precision
+            ), 
+            4326) AS geom
+    FROM wys.locations AS loc
+    WHERE length("substring"(reverse(loc.sign_name), '([0-9]{1,8})'::text)) > 3
+    ORDER BY 
+        loc.api_id, 
+        (regexp_replace(loc.sign_name, '([0-9]{5,8})'::text, ''::text)),
+        loc.start_date
+)
+
+SELECT 
+    loc.api_id,
+    loc.sign_id,
+    loc.address,
+    loc.sign_name,
+    loc.dir,
+    loc.start_date,
+    loc.serial_num,
+    loc.geom,
```

Comment:
I know this might violates one of the SQLFluff rules, but we prefer to have the geometry column the last column in tables/views for easier browsing

Reply:
I would suggest limiting your column length as per Natalie's excellent lightning talk :) 
https://www.notion.so/bditto/Fix-geom-column-length-in-pgadmin4-3457d3236eb04317974e9bacd3d03cf0?pvs=4

Reply:
By easier brwosing, I meant having the more important (meaningful) columns at the beginning and pushing geometry columns to the end. But, that's okay